ros2service/ros2service/verb/call.py

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import importlib
+
 import time
+from typing import Optional
 
 import rclpy
-from rclpy.qos import QoSPresetProfiles
+from rclpy.qos import QoSPresetProfiles, QoSProfile
```

Comment:
We prefer having one import per line:

```suggestion
from rclpy.qos import QoSPresetProfiles
from rclpy.qos import QoSProfile
```

Reply:
The source for this is our code style guide: https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#id4

it links to PEP8: https://peps.python.org/pep-0008/#imports

Reply:
oh, but PEP8 says:

> It’s okay to say this though:
> 
> ```py
> # Correct:
> from subprocess import Popen, PIPE
> ```

Reply:
feel free to leave it then